### PR TITLE
Machine Config bug batching 4.17

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.17

Issue:
N/A

Link to docs preview:
https://82560--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html

QE review:
N/A

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -1368,6 +1368,26 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
 [id="ocp-4-17-machine-config-operator-bug-fixes_{context}"]
 ==== Machine Config Operator
 
+* Previously, in a cluster that runs {product-title} 4.16 with the Telco RAN DU reference configuration, long duration `cyclictest` or `timerlat` tests could fail with maximum latencies detected above `20`us. This issue occured because the `psi` kernel command line argument was being set to `1` by default when cgroup v2 is enabled. With this release, the issue is fixed by setting `psi=0` in the kernel arguments when enabling cgroup v2. The `cyclictest` latency issue reported in link:https://issues.redhat.com/browse/OCPBUGS-34022[*OCPBUGS-34022*] is now also fixed. (link:https://issues.redhat.com/browse/OCPBUGS-37271[*OCPBUGS-37271*])
+
+* Previously, if a cluster admin creates a new `MachineOSConfig` object that references a legacy pull secret, the canonicalized version of this secret that gets created is not updated whenever the original pull secret changes. With this release, the issue is resolved.
+(link:https://issues.redhat.com/browse/OCPBUGS-34079[*OCPBUGS-34079])
```
Comment (ShaunaDiaz) [inline release_notes/ocp-4-17-release-notes.adoc:1374]:
```suggestion
(link:https://issues.redhat.com/browse/OCPBUGS-34079[*OCPBUGS-34079*])
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -1368,6 +1368,26 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
...
+	
+* Previously, a potential panic was seen in Machine Config Controller and Machine Build Controller objects if a de-reference accidentally deleted `MachineOSConfig/MachineOSBuild` to read the build status. The panic is controlled with additional error conditions to warn for allowed MachineOSConfig deletions. (link:https://issues.redhat.com/browse/OCPBUGS-33129[*OCPBUGS-33129*])
+
+* Previously, after upgrading from {product-title} 4.1 or 4.2 to version 4.15, some machines could get stuck during provisioning and never became available. This was because the `machine-config-daemon-firstboot` service was failing due to an incompatible `machine-config-daemon` binary on those nodes. With this release, the correct `machine-config-daemon` binary is copied to nodes before booting. (link:https://issues.redhat.com/browse/OCPBUGS-28974[*OCPBUGS-28974*])
+
+* Previously, if you attempted to configure on-cluster {op-system-first} image layering on a non-{op-system} node, the node would become degraded. With this fix, in this situation you will receive an error message in the node logs, but the node will not be degraded. (link:https://issues.redhat.com/browse/OCPBUGS-19537[*OCPBUGS-197537*])
```
Comment (ShaunaDiaz) [inline release_notes/ocp-4-17-release-notes.adoc:1389]:
```suggestion
* Previously, if you attempted to configure on-cluster {op-system-first} image layering on a non-{op-system} node, the node became degraded. With this fix, in this situation, an error message is produced in the node logs, but the node is not be degraded. (link:https://issues.redhat.com/browse/OCPBUGS-19537[*OCPBUGS-197537*])
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -1368,6 +1368,26 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
...
+(link:https://issues.redhat.com/browse/OCPBUGS-33913[*OCPBUGS-33913*])
+
+* Previously, if you created more than one `MachineOSConfig` object that required a canonicalized secret, only the first object would build. With this fix, the build controller handles multiple `MachineOSBuilds` that use the same canonicalized secret. (link:https://issues.redhat.com/browse/OCPBUGS-33671[*OCPBUGS-33671*])
+
+* Previously, if machine config pools (MCP) had a higher `maxUnavailable` value than the cluster's number of unavailable nodes, cordoned nodes were able to be erroneously selected as an update candidate. This fix adds a node readiness check in the node controller so that cordoned nodes are queued for an update. (link:https://issues.redhat.com/browse/OCPBUGS-33397[*OCPBUGS-33397*])
+
+* Previously, nodes could be drained twice if the node was queued multiple times in the drain controller. This behaviour might have been due to increased activity on the node object by on-cluster layering functionality. With this fix, a node queued for drain will only drain once. (link:https://issues.redhat.com/browse/OCPBUGS-33134[*OCPBUGS-33134*])
```
Comment (ShaunaDiaz) [inline release_notes/ocp-4-17-release-notes.adoc:1383]:
```suggestion
* Previously, nodes could be drained twice if the node was queued multiple times in the drain controller. This behaviour might have been due to increased activity on the node object by on-cluster layering functionality. With this fix, a node queued for drain only drains once. (link:https://issues.redhat.com/browse/OCPBUGS-33134[*OCPBUGS-33134*])
```

**Review by ShaunaDiaz — COMMENTED** [review]
skimmed through when I saw the broken BOLD; little style nits

**Review by maxwelldb — COMMENTED** [review]
IMO, Shauna's suggestions are good. The link formatting fix is a must, though. @obrown1205 

https://github.com/openshift/openshift-docs/pull/82560/files#r1777375795

**Review by jeana-redhat — APPROVED** [review]
LGTM

**Pull request merged**